static/js/drafts.js

```diff
@@ -365,6 +366,13 @@ exports.drafts_handle_events = function (e, event_key) {
     }
 };
 
+exports.restore_draft_on_shiftd = function() {
```

Comment:
I would change this to something like `restore_draft_for_hotkey`.

Reply:
Sure @showell